automated/linux/leds/test-leds2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+source "../test_format.sh"
+
+LED_TIME=1 #.5
+
+LEDs_Core=(
+	"a1_green"
+	"a1_red"
+	"a2_green"
+	"a2_red"
+)
+
+for i in {0..3}
+do
+	echo ${LEDs_Core[$i]}
+	echo 1 > /sys/class/leds/${LEDs_Core[$i]}/brightness
+	sleep $LED_TIME
+	echo 0 > /sys/class/leds/${LEDs_Core[$i]}/brightness
+done
+
```

Comment:
no checking for the test.  I think we can't do the automated test for this case, as some device to check the LED status is needed.

Reply:
We have thought about that and we still need to define what hardware will be used.
Since no user is needed to continue with the process, I think it is possible to add this test. As soon as the hardware is available, it should be modified. That's why there is a "TODO" at the end of the script... This shows that the test is in progress.

Reply:
LGTM